README.md

```diff
@@ -46,6 +47,81 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script does deploy a simple order market maker order and a relevant
+```
+// Get token id of DAI
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa' --network rinkeby
+
+//export the result
```

Comment:
```suggestion
// Export the resulting token ID
```

I would simply change the first comment to something like  "Fetch the token ID of DAI and export the result"